internal/expr/expr_test.go

```diff
@@ -449,6 +449,60 @@ func (s *ExprSuite) TestExprPkg(c *C) {
 	}
 }
 
+func (s *ExprSuite) TestInsertInputParser(c *C) {
+	tests := []struct {
+		summary        string
+		query          string
+		expectedParsed string
+	}{{
+		summary:        "insert asterisk",
+		query:          "INSERT INTO person (*) VALUES ($Address.street, $Person.*, $M.team)",
+		expectedParsed: "[Bypass[INSERT INTO person ] AsteriskInput[[*] [Address.street Person.* M.team]]]",
```

Comment:
It would make generating the SQL a lot more confusing, it would no longer be linear since the `Bypass[VALUES ]` would come in the middle of `AsteriskInput`. We could change the way things are printed for the parser tests so that the `VALUES` is actually printed, i.e. [Bypass[INSERT INTO person ] AsteriskInput[[*] VALUES [Address.street Person.* M.team]]]

Reply:
Yeah I like your suggestion a lot more. Either that or keep it like it is implemented now, the VALUES might be clearer when reading tests as you point out.